tests/test_loader_doc_strings.py

```diff
@@ -16,65 +16,60 @@
 # along with Galaxy.  If not, see <http://www.apache.org/licenses/>.
 
 import json
+import os
 import pytest
+import shutil
+import tempfile
 from unittest import mock
 
-from galaxy_importer import constants
 from galaxy_importer import loaders
 
 
-ANSIBLE_DOC_OUTPUT = """
-    {"my_sample_module": {
-        "doc": {
-            "description": ["Sample module for testing."],
-            "short_description": "Sample module for testing",
-            "version_added": "2.8",
-            "options": {
-                "exclude": {
-                    "description": ["This is the message to send..."],
-                    "required": "true"
-                },
-                "use_new": {
-                    "description": ["Control is passed..."],
-                    "version_added": "2.7",
-                    "default": "auto"
-                }
-            }
-        },
-        "examples": null,
-        "metadata": null,
-        "return": {
-            "message": {
-                "description": "The output message the sample module generates"
-            },
-            "original_message": {
-                "description": "The original name param that was passed in",
-                "type": "str"
-            }
-        }
-    }}
-"""
-
-
 @pytest.fixture
 def doc_string_loader():
     return loaders.DocStringLoader(
-        content_type=constants.ContentType.MODULE.value,
-        path=None,
-        fq_name='my_namespace.my_collection.my_sample_module')
+        path='/tmp_dir/tmp123/ansible_collections/my_namespace/my_collection',
+        fq_collection_name='my_namespace.my_collection')
 
 
 def test_init_loader(doc_string_loader):
-    assert doc_string_loader.content_type == 'module'
-    assert doc_string_loader.fq_name == 'my_namespace.my_collection.my_sample_module'
+    assert doc_string_loader.fq_collection_name == 'my_namespace.my_collection'
+
+
+def test_get_plugins(doc_string_loader):
+    temp_dir = tempfile.mkdtemp()
+    with open(os.path.join(temp_dir, '__init__.py'), 'w'):
+        pass
+    with open(os.path.join(temp_dir, 'should_be_ignored.txt'), 'w'):
+        pass
+    with open(os.path.join(temp_dir, 'my_module.py'), 'w'):
+        pass
+    plugins = doc_string_loader._get_plugins(temp_dir)
+
+    assert plugins == ['my_namespace.my_collection.my_module']
+    shutil.rmtree(temp_dir)
```

Comment:
For this case, I might cleanup the temp_dir before doing any asserts. Otherwise failed tests could leave debris behind.

May also be a place where some of the pytest temp dir fixtures could be used.